packages/core/src/test/testUtil.ts

```diff
@@ -185,6 +185,10 @@ export async function createTestWorkspace(
          * optional file name suffix
          */
         fileNameSuffix?: string
+        /**
+         * optional additional files
+         */
+        additionalFiles?: Array<{ fileName: string; fileContent: string }>
```

Comment:
Instead of making this does it make sense to use the TestFolder class? Then you would just do something like:

```
const folder = TestFolder.create()
folder.write('devfile.yaml', '')
folder.write('file.md', 'test content')
```

Reply:
ok let me do that. hm, I thought the `TestFolder` didn't work outside of `testUtil.ts`, i was wrong, let me update, thanks!